docs/resource.md

```diff
@@ -52,42 +51,6 @@ url: https://cloud-images.ubuntu.com/releases/16.04/release/ubuntu-16.04-server-
 - `file`: a local file path
 - `compression`: optional field to specify decompress method.  Currently, "gzip" and "bzip2" are supported.
 
-DataFolder resource
```

Comment:
@ymmt2005 
Although I removed DataFolder resource, [CKE's mtest](https://github.com/cybozu-go/cke/blob/master/mtest/cluster.yml#L8) is sharing necessary files to guests by listing all files with DataFolder resource.
I think we can do this alternatively by putting the necessary files in a folder in advance and specifying the folder path, but it would be a bit bother. What do you think?

Reply:
Yes, we can.  DataFolder was just for convenience.

Reply:
OK, I'll remove it then.